### ISSUE TITLE
Support multiple exchanges connected to the same port

### ISSUE THREAD

**Issue by danionita** [danionita/e3tools#89]
![image](https://cloud.githubusercontent.com/assets/11060388/21310598/6153f910-c5e3-11e6-8100-06128133b52c.png)

Bob, if you get around to this, let me know, as it requires some adaptations in my part of the code to function.

**Comment by bobismijnnaam** [comment]
It depends... Does it mess with second_connects_ve et al.? :p

**Comment by danionita** [comment]
second_connects_ve et al only appear when there are exchanges on both sides of the interface.

In this case, the Value Port with multiple exchanges would simply have mulitple vp_in_connects_ve or vp_out_connects attributes, depending on the direction of port. 

Of course,  when both cases are present (there are exchanges on both sides _and_ multiple exchanges connected to the same port on one side), then you might end up with multiple vp_first_connects_ve or vp_second_connects_ve .

Is this workable?

**Comment by bobismijnnaam** [comment]
Well, just tested it with this example:

![image](https://cloud.githubusercontent.com/assets/3082344/21641610/3e9fd4a6-d27d-11e6-8096-b0e1a2b3ed18.png)

The RDF export works, and the fraud generation also works, it seems. Maybe @danionita  can check if it gives plausible analysis results if you give it meaningful valuation attributes and such?

Also checked it with this model:

![image](https://cloud.githubusercontent.com/assets/3082344/21641852/bf159192-d27e-11e6-9133-5add74ee00ba.png)

It gives an error that a start stimuli entity was found while traveling downstream. Makes sense in some way; it starts at a start stimuli in the top left, and then branches out from the center value interface. Of course it's going to end up at the start stimuli in the top right. I don't immediately see a clear and elegant solution to this (as this is an ambiguity in the e3value ontology, I think). I think we can only document this for the user, and say that structures like these are not allowed (i.e. decide that multiple value exchanges are only allowed on the "downstream" side of a value interface).

**Comment by bobismijnnaam** [comment]
Also checked the code somewhat. It seems that in most places the exporter doesn't care about the amount of value exchanges, but just checks them all and handles them all in the same way. So that should be fine for now I guess.

**Comment by danionita** [comment]
I fail to see why the second example, containing the "Cross" is ambiguous. The flow should not travel from the top left to the top right corner but rather first from the top left to the two bottom corners and then from the top-right to the two bottom corners.

Multiple transfers arriving at (or departing from) the same port have the same meaning as if those transfers would merge (or split) via an OR-gate before  the port. See also fig 27 on page 26 of the e3manual: http://e3value.few.vu.nl/docs/misc/manual_version2.pdf)
![image](https://cloud.githubusercontent.com/assets/11060388/21768951/ea8665b8-d684-11e6-8155-7c847bd5a4d0.png)

This, of course, implies that one should be able to (1) manually group transfers into transactions and (2) define FRACTION attributed to these transactions, just like in the old editor. Somehow, we completely forgot about the transactions editor (which was supposed to handle cases where transfers could not be grouped into transactions automatically), so I have created Issue #90. 
(We should also allow dependency paths to cross inner-actor borders, so I created issue #91 too).

Your code should be able generate an RDF from both the models you show. However, for the RDF to be _usable_ by the NCF module from the second "crossed" model, it would need to contain manually defined transactions and fractions.
